### DISCUSSION TITLE
Disable ScrolledText after creation

### DISCUSSION THREAD

**Discussion by impubes** [israel-dryer/ttkbootstrap#201] [Q&A] [answered]
Hello
How can i disable (change the state of) ScrolledText after creation like;
```python
st = ScrolledText(root, padding=5, height=15, autohide=True)
st.configure(state="disabled")
```

Above code gives me the following error:

```
.......
tkinter.TclError: unknown option "-state"
```
Thank you

**Comment by israel-dryer** [comment, marked as answer]
I think I should probably override the configure method to route certain configurations to the internal text widget.

However, for now, you can access the internal text widget as below and this should do what you need. I'll add this to the list of improvements to make on the next release.
```python
st._text.configure(state='disabled')
```

**Reply by impubes** [reply]
> I think I should probably override the configure method to route certain configurations to the internal text widget.
> 
> However, for now, you can access the internal text widget as below and this should do what you need. I'll add this to the list of improvements to make on the next release.
> 
> ```python
> st._text.configure(state='disabled')
> ```

Thank you for your quick reply @israel-dryer :)
